app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb

```diff
@@ -33,6 +33,12 @@
               description: "As a design and technology teacher, you'll help pupils to turn their ideas into reality.",
               path: "/life-as-a-teacher/explore-subjects/design-and-technology",
             )) %>
+              <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Drama",
+              description: "As a drama teacher, you'll inspire pupils to unlock their inner creativity and take bold risks.",
```

Comment:
Not sure about bold risks, maybe something about building confidence instead? Or leave as is!